.travis.yml

```diff
@@ -0,0 +1,120 @@
+# Use the R language default, as travis always has Python
+language: python
```

Comment:
But I should have changed the comment after switching from R to python.  I'll do so.